src/middlewared/middlewared/plugins/etc.py

```diff
@@ -195,6 +195,8 @@ class EtcService(Service):
         ],
         'ssh': [
             {'type': 'mako', 'path': 'local/ssh/sshd_config'},
+            {'type': 'mako', 'path': 'local/users.oath'},
```

Comment:
I think you dont want users.oath file to have group/others permissions?

Reply:
By default it has 0600 perms and those are exactly what we require but i should make it explicit here.

Reply:
It should not be 0600 by default, there is likely a bug somewhere changing umask